sig/transaction/README.md

```diff
@@ -1,13 +1,35 @@
 # Transaction Special Interest Group
 
-Covers transaction related work in TiKV, including improve transaction in TiKV and explore new transaction model.
+The transactions special interest group (SIG-transaction) are a group of people interested in transactions in distributed databases. We have a focus on transactions in TiKV and TiDB, but discuss academic work and other implementations too.
 
-- Repository: https://github.com/tikv/sig-transaction (Most useful resource, contains lots of information about the SIG and transactions)
```

Comment:
Can we keep a link to the repo near the top please?

Also we should keep links to membership.md and workflow_zh.md somewhere. (people.md in the repo is a more informal docu, meant to have people's bios, etc. membership.md is the definitive and machine-readable list).

Reply:
Fixed